src/pages/ShoppingCart/utils/getCustomerServiceLink.ts

```diff
@@ -0,0 +1,46 @@
+import { ShoppingCartProduct } from './concatProductsAndQuantity';
+
+interface GetWhatsappLinkProps {
+  phone: string;
+  message: string;
+}
+
+export const getWebWhatsappLink = ({
+  phone,
+  message,
+}: GetWhatsappLinkProps) => {
+  const phoneWithoutSpaces = phone.replaceAll(' ', '');
+  const url = `https://wa.me/${phoneWithoutSpaces}?text=${encodeURIComponent(
+    message
+  )}`;
+  return url;
+};
+
+export const getInvoiceText = (shoppingCartProducts: ShoppingCartProduct[]) => {
+  let allProductsTotal = 0;
+
+  let invoice = shoppingCartProducts.reduce((acc, { product, quantity }) => {
+    const { name, base_price_sol } = product;
+    const total = quantity * base_price_sol;
```

Comment:
Please cast to numbers before, for example 
```suggestion
    const total = Number(quantity) * Number(base_price_sol);
```

Reply:
I think another method is using `+`, like `+quantity * +base_price_sol` but maybe is a good idea to test this approach

Reply:
Yes, I will check

Reply:
It works, but I think that trick to cast numbers is a little weird hehe, maybe new developers don't understand what is `+quantity * +base_price_sol`, maybe it's so new? Hmm

@e-azocar what do you think?

Reply:
What about using Number()?

Reply:
Of course

Reply:
Applied, please check